pkg/symbol/addr2line/symtab.go

```diff
@@ -106,14 +109,54 @@ func (lnr *SymtabLiner) PCToLines(addr uint64) (lines []profile.LocationLine, er
 // to facilitate searching.
 func symtab(objFile *elf.File) ([]elf.Symbol, error) {
 	syms, sErr := objFile.Symbols()
+	dynSyms, dErr := objFile.DynamicSymbols()
+
+	var pltSymbols []elf.Symbol
```

Comment:
♻️ It's better to create a function to extract pltSymbols. It's already contained. It would make the codebase more readable.